docs/utils/gen_docs.py

```
@@ -137,10 +127,16 @@ def get_compatible_devices(obj: PrunaAlgorithmBase) -> str:
     """Get the compatible devices of a Pruna algorithm."""
     compatible_devices = []
     for device in obj.compatible_devices():
-        if device == "cpu":
-            compatible_devices.append("CPU")
-        elif device == "cuda":
-            compatible_devices.append("GPU")
+        name_map = {
+            "cpu": "CPU",
+            "cuda": "CUDA",
+            "mps": "MPS",
+            "accelerate": "Accelerate distributed",
+        }
+        if device in name_map:
+            compatible_devices.append(name_map[device])
+        else:
+            compatible_devices.append(device)
```

Perhaps we should just let it fail to enforce people to update the device map?